bin/pasta_show_maintainers.py

```diff
@@ -0,0 +1,165 @@
+    # Empty file:
+    except:
+        return [0, 0]
+
+all_filenames = [os.path.join(r, item)[23:] for r, _, f in os.walk('./resources/linux/repo') for item in f]
+
+file_lengths = dict()
+for filename in all_filenames:
+    file_lengths[filename] = file_len('./resources/linux/repo/' + str.strip(filename))
+
+def get_entry_counts(all_maintainers, filenames = all_filenames):
+    subsystem_dirs = dict()
+    counts_by_entry = dict()
+    for filename in filenames:
+        subsystems = all_maintainers.get_subsystems_by_file(filename)
+        for subsystem in subsystems:
+            subsystem_dirs[subsystem] = subsystem_dirs.get(subsystem, set())
+            subsystem_dirs[subsystem].add(''.join(re.findall('[a-zA-Z0-9_]+/', filename)))
+            counts_by_entry[subsystem] = np.add(counts_by_entry.get(subsystem, [0, 0]), file_lengths[filename])
+    return OrderedDict(sorted(counts_by_entry.items(), key=lambda x: x[1][1], reverse=True)), subsystem_dirs
+
+def get_maintainer_counts(all_maintainers, filenames = all_filenames):
+    maintainers_subs = dict()
+    counts_by_maintainer = dict()
+    for filename in filenames:
+        subsystems = all_maintainers.get_subsystems_by_file(filename)
+        for subsystem in subsystems:
+            maintainers = all_maintainers.get_maintainers(subsystem)
+            for maintainer in flatten(maintainers):
+                if not (type(maintainer) == str or maintainer[0] == '' or maintainer[1] == ''):
+                    maintainers_subs[maintainer[0]+' <'+maintainer[1]+'>'] = maintainers_subs.get(maintainer[0]+' <'+maintainer[1]+'>', set())
+                    maintainers_subs[maintainer[0]+' <'+maintainer[1]+'>'].add(subsystem)
+                    counts_by_maintainer[maintainer[0]+' <'+maintainer[1]+'>'] = np.add(counts_by_maintainer.get(maintainer[0]+' <'+maintainer[1]+'>', [0, 0]), file_lengths[filename])
+            
+    return OrderedDict(sorted(counts_by_maintainer.items(), key=lambda x: x[1][1], reverse=True)), maintainers_subs
+
+def get_small(title, counts):
+    results = list()
+    results.append([title, 'LoC'])
+    for entry in counts:
+        results.append([entry, counts[entry][0]])
+    return results
+
+def get_maintainers(config, sub, argv):
+    
+    parser = ArgumentParser(description='d')
+
+    parser.add_argument('maintainers', nargs=2, type=bool)
+    parser.add_argument('query', type=str, help='Group by subsystems or maintainers')
+    parser.add_argument('--smallstat', action='store_true', help='Simple view')
+    parser.add_argument('--outfile', type=str, help='Output to a csv file')
+    parser.add_argument('--file', type=str, help='Select MAINTAINERS file')
+    parser.add_argument('--filter', type=str, help='Filter by file list')
+
+    args = parser.parse_args()
+
+    if args.file!= None:
+        all_maintainers_text = file_to_string(args.file)
+    else:
+        all_maintainers_text = file_to_string('./resources/linux/repo/MAINTAINERS')
```

Comment:
is there something like a default function in python default(args.file, b) means args.file if that is not None else b.

Reply:
Then this can be expressed less verbose.